CHANGES.md

```diff
@@ -1,3 +1,6 @@
+1.0.1
+* Added get all configurations
```

Comment:
I think it also has template related features which were not there earlier. You can mention those as well here.